weblab_cg/_generator_chaste_cg.py

```diff
@@ -0,0 +1,627 @@
+#
```

Comment:
This looks pretty serious. Nice!

Reply:
Should I change my generator to `weblab_generator` then? If they're not sharing any code?

Reply:
could do, up to you really, but we're sharing quite a lot still (hence the sub classing)